src/components/DistanceMapView/index.android.tsx

```diff
@@ -30,7 +27,7 @@ function DistanceMapView(props) {
                 }}
             />
             {!isMapReady && (
-                <View style={StyleUtils.combineStyles(styles.mapViewOverlay, props.overlayStyle)}>
+                <View style={StyleSheet.flatten([styles.mapViewOverlay, overlayStyle])}>
```

Comment:
Why do we need `StyleSheet.flatten` here? Because `overlayStyle` can be an array? RN supports nested arrays in style, so it should work without `StyleSheet.flatten` as well, wdyt?

Reply:
Also, I think we should be focused on solving the issue in the issue-fixing PRs and keep the unrelated changes to a minimum. TS migrations should migrate to TypeScript. It makes it easier to validate their correctness.

Unless there's a specific TypeScript-related reason for this change, I agree that it shouldn't be included.

Reply:
I guess these updates are related to type updates of the `overlayStyle` param, so it can go with this PR

Reply:
Mhm. But what are you suggesting in this thread? To replace the discussed call expression with `[styles.mapViewOverlay, overlayStyle]`?

Reply:
@VickyStash I made this change because the type for the `StyleUtils.combineStyles` was `ViewStyle | TextStyle | ImageStyle` while the `overlayStyle` was of type `StyleProp<ViewStyle>` so the type was not working out. But, if we want I think it is safe to replace this with a simple array as well

Reply:
Made the change here: https://github.com/Expensify/App/pull/33553/commits/804b01c65121b14689c46be55f399fc3a1b3b9c3